src/test/polymer/polymer-element-scanner_test.ts

```diff
@@ -116,6 +116,17 @@ suite('PolymerElementScanner', () => {
         ['all', 'Object']
       ]);
 
+      assert.deepEqual(features[0].attributes.map(p => [p.name, p.changeEvent]), [
+        ['a', undefined],
+        ['b', undefined],
+        ['c', undefined],
+        ['d', undefined],
+        ['e', 'e-changed'],
+        ['f', undefined],
+        ['g', undefined],
```

Comment:
Looks like the test is finding `['h', undefined],` as well? Which is kinda weird because I don't see an 'h' property defined anywhere